### PR TITLE
KOGITO-5085: Replace Blob* classes with elemental2 equivalent

### PR DESCRIPTION
**Pull request by treblereel**
JIRA: https://issues.redhat.com/browse/KOGITO-5085

### CODE REVIEW

**Comment by treblereel** [comment]
do not merge it for now

File: uberfire-extensions/uberfire-commons-editor/uberfire-commons-editor-client/src/main/java/org/uberfire/ext/editor/commons/client/file/exports/svg/SvgFileExport.java [hunk 1]
```diff
@@ -37,6 +40,10 @@ protected SvgFileExport(BiConsumer<Blob, String> saveAs) {
 
     @Override
     protected Optional<Blob> getContent(IContext2D entity) {
-        return Optional.of(BlobImpl.create(entity.getSerializedSvg()));
+        Object value = entity.getSerializedSvg();
+        final Blob blob = new Blob(new ConstructorBlobPartsArrayUnionType[] {ConstructorBlobPartsArrayUnionType.of(value) });
+
+        //final Blob blob = new Blob(new ConstructorBlobPartsArrayUnionType[]{ConstructorBlobPartsArrayUnionType.of(entity.getSerializedSvg())});
```
Comment (tiagobento) [inline uberfire-extensions/uberfire-commons-editor/uberfire-commons-editor-client/src/main/java/org/uberfire/ext/editor/commons/client/file/exports/svg/SvgFileExport.java:46]: Maybe remove that comment?

Comment (treblereel) [reply uberfire-extensions/uberfire-commons-editor/uberfire-commons-editor-client/src/main/java/org/uberfire/ext/editor/commons/client/file/exports/svg/SvgFileExport.java:46]: i want to do few additional checks

**Comment by treblereel** [comment]
@tiagobento looks like it's good to go

**Comment by tiagobento** [comment]
@tomasdavidorg Hi Tomas, can you please verify this PR?

**Review by tomasdavidorg — COMMENTED** [review]
Just minor formatting notes. I will continue with the review next week.

**Review by tomasdavidorg — APPROVED** [review]

**Review by tiagobento — APPROVED** [review]

**Pull request merged**